frontend/app/views/spree/checkout/_payment.html.erb

```diff
@@ -59,7 +59,15 @@
     <p class='field' data-hook='coupon_code'>
       <%= form.label :coupon_code %><br />
       <%= form.text_field :coupon_code %>
+      <button type="button" class="button" id="coupon-code-apply-button">
+        <%= Spree.t(:apply_code) %>
+      </button>
     </p>
+    <div id='coupon_status'
```

Comment:
Ah, I went about to change it, but it turns out that `<div>`s [can't be nested](http://stackoverflow.com/questions/8397852/why-p-tag-cant-contain-div-tag-inside-it) inside `<p>` tags.

Funny that I had tried to place it inside the `<p>` on my initial go, had discovered that it doesn't work, but had forgotten about it, and was trying it again ..  so bad ..